Formula/svt-hevc.rb

```diff
@@ -0,0 +1,31 @@
+class SvtHevc < Formula
+  desc "Scalable Video Technology for HEVC Encoder"
+  homepage "https://01.org/svt"
+  url "https://github.com/OpenVisualCloud/SVT-HEVC/archive/v1.4.3.tar.gz"
+  sha256 "08bf2b1075609788194bf983d693ab38ed30214ec966afa67895209fd0bf2179"
+  head "https://github.com/OpenVisualCloud/SVT-HEVC.git"
+
+  depends_on "cmake" => :build
+  depends_on "yasm" => :build
+
+  resource "akiyo_qcif.y4m" do
+    url "https://media.xiph.org/video/derf/y4m/akiyo_qcif.y4m"
+    sha256 "df88d83cbf6d99f3ec41f2c1fd2e67665d2a71ff8caa08f8b6bc46bf4567ea2e"
+  end
+
+  def install
+    resource("akiyo_qcif.y4m").stage(pkgshare.to_s)
```

Comment:
If we need this file only for testing purposes, there is no reason to install it.
You can use `resource("akiyo_qcif.y4m").stage(testpath)` in test block

Reply:
Thanks for suggestion!